### PR TITLE
rename 'divided difference operator'

### PR DESCRIPTION
**Pull request by jacksonwalters**
The action of $e_i^{(p)}$ on Fock space corresponding to pair `(i,p)` is given by $e_i^{(p)} := e_i^{p} / [p]_q$.

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] The description explains in detail what this PR is about.
- [x] I have linked a relevant issue or discussion.
- [x] I have created tests covering the changes.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: src/sage/algebras/quantum_groups/fock_space.py [hunk 1]
```diff
@@ -754,8 +754,8 @@ def N_left(la, x, i):
 
             def e(self, *data):
                 r"""
-                Apply the action of the divided difference operator
-                `e_i^{(p)}` on ``self``.
+                Apply the action of the divided power operator
+                `e_i^{(p)}` = e_i^{p} / [p]_q on ``self``.
```
Comment (tscrim) [inline src/sage/algebras/quantum_groups/fock_space.py]:
```suggestion
                `e_i^{(p)} = e_i^{p} / [p]_q` on ``self``.
```

**Review by tscrim — CHANGES_REQUESTED** [review]
The `f` operator should also have a similar change.

**Review by jacksonwalters — COMMENTED** [review]
make analogous change for f

**Review by tscrim — CHANGES_REQUESTED** [review]
Thank you. This is good except for the change in `pkgs/sagemath-standard/setup.cfg`. Just revert that and it will be a positive review.

**Review by tscrim — APPROVED** [review]
@jacksonwalters Thank you.

@mkoeppe Matthias, something seems to be strange with this branch. Can you quickly fix the weirdness with the `setup.cfg`? I don't have super-push powers. This is a positive review otherwise.

**Comment by mkoeppe** [comment]
I've squashed it away

**Comment by tscrim** [comment]
@mkoeppe Thank you very much!

**Comment by jacksonwalters** [comment]
sorry, is there anything I can do?

**Comment by tscrim** [comment]
This should fix it. @jacksonwalters Please don't push to this branch (at least until this is merged). For future reference, it is good to have the branches have unique names; still somewhat short but related to the issue/PR in question.

**Comment by jacksonwalters** [comment]
@tscrim Sounds good, thank you. I went ahead and deleted my local copy and re-cloned the repo. Git is no longer asking me to push changes. I will leave it alone.

**Pull request merged**